src/useCases/generateEmailVerificationUrl.ts

```diff
@@ -26,6 +27,12 @@ export default async (
     return undefined
   }
 
+  const resetFailedPasswordAttemptsResult = await setFailedPasswordAttempts(connection, emailAddress, 0)
```

Comment:
I wonder if updating `failed_password_attempts ` can also be done in the query in`storeVerificationCode` use case. The reason I'm saying that is that `failed_password_attempts` is for the generated verification code so if we are updating the user record for the verification code then we should probably set `failed_password_attempts ` to 0 at the same time.

Reply:
Sure, will do that